src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SearchJqlStarAction.kt

```diff
@@ -0,0 +1,37 @@
+package com.atlassian.performance.tools.jiraactions.api.action
+
+import com.atlassian.performance.tools.jiraactions.api.SEARCH_WITH_JQL_A_STAR
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.IssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.JqlMemory
+import com.atlassian.performance.tools.jiraactions.api.observation.SearchJqlObservation
+import com.atlassian.performance.tools.jiraactions.api.page.IssueNavigatorPage
+import java.util.function.Predicate
+import javax.json.JsonObject
+
+class SearchJqlStarAction(
+    private val jira: WebJira,
+    private val meter: ActionMeter,
+    private val jqlMemory: JqlMemory,
+    private val issueKeyMemory: IssueKeyMemory
+) : Action {
+    override fun run() {
+        val jqlQuery = jqlMemory.recallByTag(Predicate { s -> s == "GENERIC_WIDE" })!!
```

Comment:
I think we need to account for `null` here. What if there are no such queries?

Reply:
This query is [hardcoded](https://github.com/atlassian/jira-actions/blob/master/src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/adaptive/AdaptiveJqlMemory.kt#L25).